Hydra/ApiDocumentationBuilder.php

```diff
@@ -134,10 +142,10 @@ public function getApiDocumentation()
                     'hydra:property' => [
                         '@id' => ($iri = $attributeMetadata->getIri()) ? $iri : sprintf('#%s/%s', $shortName, $attributeName),
                         '@type' => $type,
-                        'rdfs:label' => $attributeName,
+                        'rdfs:label' => $convertedName,
```

Comment:
`rdf:label` and `hydra:title` are indeed descriptions for human readers. You can let it as is.

For Nelmio, it will need a patch in that bundle directly.